airflow/config_templates/default_airflow.cfg

```diff
@@ -591,6 +591,11 @@ json_format = False
 # Log fields to also attach to the json output, if enabled
 json_fields = asctime, filename, lineno, levelname, message
 
+[elasticsearch_configs]
+
+use_ssl = False
+verify_certs = False
```

Comment:
@dimberman wondering if we should mirror the defaults that the es client uses here so we don't change any behavior. `verify_certs` is `True` by default here: https://github.com/elastic/elasticsearch-py/blob/master/elasticsearch/connection/http_requests.py#L47-L48

Reply:
@schnie good catch, thought they both defaulted to false